app/workers/articles/update_main_image_background_hex_worker.rb

```diff
@@ -0,0 +1,15 @@
+module Articles
+  class UpdateMainImageBackgroundHexWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority, retry: 10
```

Comment:
@mstruve what do you think about this priority?

Reply:
I do think after a user saves their article they will want to see it, with proper colors, right away so this seems like the right queue. We can always adjust later.